frontend/src/app/(tabs)/resources.tsx

```diff
@@ -62,7 +62,9 @@ export default function Resources() {
     <View style={styles.container}>
       <Mascots style={styles.logo} />
       <Text style={styles.title}>Psyches of Color</Text>
-      <Text style={styles.text}>slogan/mindful tip goes here</Text>
+      <Text style={styles.text}>
+        Celebrate one thing you handled well today, no matter how small.
```

Comment:
Can you add some horizonal margins here please? The text goes all the way to the left/right edges of the screen for me. 20px should be fine